src/App.js

```diff
@@ -0,0 +1,67 @@
+import ListPage from "./page/ListPage.js";
+import EditorPage from "./page/EditorPage.js";
+import HomePage from "./page/HomePage.js";
+import { initRouter } from "../util/router.js";
+
+export default function App({ $target }) {
+  const $documentListContainer = document.createElement("div");
+  $documentListContainer.classList = "list-container";
+  const $EditorContainer = document.createElement("div");
+  $EditorContainer.classList = "editor-container";
+
+  $target.appendChild($documentListContainer);
+  $target.appendChild($EditorContainer);
+
+  const editorPage = new EditorPage({
+    $target: $EditorContainer,
+    initialState: {
+      selectedID: "main",
+      selectedDocument: {
+        title: "",
+        content: "",
+      },
+    },
+    onListRender: () => {
+      listPage.fetchDocumentsList();
+    },
+  });
+
+  const listPage = new ListPage({
+    $target: $documentListContainer,
+    initialState: { documentsTree: [], isLoading: true },
+  });
+
+  const homePage = new HomePage({
+    $target: $EditorContainer,
+  });
```

Comment:
```
export default function HomePage({ $target }) {
  const $home = document.createElement("div");
  const $span = document.createElement("div");
  $span.innerHTML = `Hober's Notion`;

  $home.classList = "home-page";

  const $image = document.createElement("img");
  $image.src = "../img/notion.PNG";
  $home.appendChild($span);
  $span.appendChild($image);

  const render = () => $target.appendChild($home);

  return {
    render,
  };
}

```

```suggestion
  const homePage = HomePage({
    $target: $EditorContainer,
  });
```

클래스를 사용하지 않고 있다면 굳이 생성자 함수를 쓰지 않고도 구현이 가능하니 참고하세요